### ISSUE TITLE
Add a Google like theme to susper.com

### ISSUE THREAD

**Issue by mariobehling** [fossasia/susper.com#4]
* [ ] Implement Google theme (without logo)
* [ ] Implement moving up of search area to top left on typing start similar to Google

**Comment by sj29-innovate** [comment]
Do i need to change the SUSPER image above the search box so that it looks like google with different colors ?

**Comment by mariobehling** [comment]
> Do i need to change the SUSPER image above the search box so that it looks like google with different colors ?

No, you don't. This is about the theme (colors, font, design, top bar, bottom bar) not the specific logo.

**Comment by sj29-innovate** [comment]
For the search bar movement do think the index page should be redirected to the other one or i should do it without redirection ?

**Comment by mariobehling** [comment]
@sj29-innovate It should be using Javascript. Please compare to http://loklak.fossasia.org

**Comment by sj29-innovate** [comment]
Please let me know if this looks good.
![moved](https://cloud.githubusercontent.com/assets/15885458/21059633/6c007af4-be69-11e6-99ff-6e762f05dec9.png)

**Comment by sj29-innovate** [comment]
I have made some changes .Please have a look.If it looks good to you then i will commit them.
![screenshot 2](https://cloud.githubusercontent.com/assets/15885458/21083118/5187e8d6-c00f-11e6-844f-c001fadba40a.png)
![screenshot 3](https://cloud.githubusercontent.com/assets/15885458/21083119/518cf3d0-c00f-11e6-87f5-cd220ad43688.png)
![screenshot 4](https://cloud.githubusercontent.com/assets/15885458/21083120/51929f6a-c00f-11e6-8e9a-e5a6bb22c5ae.png)

**Comment by sj29-innovate** [comment]
Mario can you please have a look on the above screenshots and provide a feedback.
I will commit the changes as per your feedback.

**Comment by sj29-innovate** [comment]
@hemantjadon  i was trying to squash the commits but its giving me error at the point where my previously merged request was reverted.
As of now the i have fixed the issue and the latest changes are in the last commit which is "Fixing Issue #4"

**Comment by mariobehling** [comment]
It looks good to me. Let's iterate from here. 

Please merge if ok.
